spec/requests/posts_spec.rb

```diff
@@ -1,35 +1,35 @@
 require 'rails_helper'
 
-RSpec.describe 'PostsController', type: :request do
-  let(:author) { create(:author) }
+RSpec.describe 'Posts', type: :request do
+  it 'redirects to the Posts page and checks the correct placeholder text and status' do
+    get '/users/1/posts'
 
```

Comment:
Posts are failing because some specs are vulnerable eg: 

```
  it 'does not render a different template' do
    get '/users/1/posts'
    expect(response).to_not render_template(:show)
  end
```

Here you are hardcoding the id of the user, but what if user has different id when created, it is recommended to create user with before action or let keyword and pass its id dynamically 👍🏻